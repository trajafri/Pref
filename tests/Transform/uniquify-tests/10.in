(define f (lambda (x) (if (zero? x) 1 (* x (f (- x 1))))))

(lambda (x x x x) x)
