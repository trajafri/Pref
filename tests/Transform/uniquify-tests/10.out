(define f.0 (lambda (x.0) (if (zero? x.0) 1 (* x.0 (f.0 (- x.0 1))))))

(lambda (x.1 x.2 x.3 x.4) x.4)
